libs/utils/src/zid.rs

```diff
@@ -218,7 +218,7 @@ impl ZTenantTimelineId {
 
 impl fmt::Display for ZTenantTimelineId {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        write!(f, "{}-{}", self.tenant_id, self.timeline_id)
+        write!(f, "{}/{}", self.tenant_id, self.timeline_id)
```

Comment:
What this change is about?

Reply:
Ah, it is for easier copypaste of log pieces into various places, for example safekeeper timeline status endpoint accepts {tenant_id}/{timeline_id}, not {tenant_id}-{timeline_id}.